build.gradle

```diff
@@ -3,9 +3,14 @@ allprojects {
     apply plugin: 'java'
     apply plugin: 'idea'
     apply plugin: 'checkstyle'
+    apply plugin: "maven"
+
+    group = "com.hazelcast.jet"
```

Comment:
I am not sure if that's the best group name. perhaps it should have the word 'contrib'. it's best to see what others are doing.